modules/monitoring-about-specifying-limits-and-requests-for-monitoring-components.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-specifying-limits-and-requests-for-monitoring-components_{context}"]
+= About specifying limits and requests for monitoring components
+
+You can configure resource limits and request settings for core platform monitoring components and for the components that monitor user-defined projects, including the following:
+
+* Alertmanager (for core platform monitoring and for user-defined projects)
+* kube-state-metrics
+* monitoring-plugin
+* node-exporter
+* openshift-state-metrics
+* Prometheus (for core platform monitoring and for user-defined projects)
+* Prometheus Adapter
+* Prometheus Operator and its admission webhook service
+* Telemeter Client
+* Thanos Querier
+* Thanos Ruler
+
+By defining resource limits, you limit a container's resource usage, which prevents the container from exceeding the specified maximum values for CPU and memory resources.
+
+By defining resource requests, you specify that a container should only be scheduled on a node that has enough CPU and memory resources available to match the requested resources.
```

Comment:
```suggestion
By defining resource requests, you specify that a container can be scheduled only on a node that has enough CPU and memory resources available to match the requested resources.
```
- IBM Style lists "should" as a use-with-caution word, so I replaced it with "can". If that's not accurate, feel free to ignore my suggestion.
- I moved "only" closer to the phrase it modifies. I assume it's modifying "a node that has enough CPU and memory resources available" and not "scheduled".

Reply:
Perfect. Thanks you!